uperf_wrapper/uperf_wrapper.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import os
+
+from uperf_wrapper.trigger_uperf import Trigger_uperf
+
+
+class uperf_wrapper():
+
+    def __init__(self, parser):
+        parser.add_argument(
+            '-w', '--workload', nargs=1,
+            help='Provide XML workload location')
+        parser.add_argument(
+            '-r', '--run', nargs=1,
+            help='Provide the iteration for the run')
+        parser.add_argument(
+            '--resourcetype', nargs=1,
+            help='Provide the resource type for this uperf run - pod/vm/baremetal')
+        self.args = parser.parse_args()
+
+        self.args.uuid = ""
```

Comment:
uuid is set here to an empty string, but not updated wth the uuid from environment variable